inst/templates/app_structure/app/main.R

```diff
@@ -1,13 +1,17 @@
 box::use(
-  shiny[bootstrapPage, moduleServer, NS, renderText, tags, textOutput],
+  shiny[bootstrapPage, div, moduleServer, NS, renderText, tags, textOutput],
 )
 
 #' @export
 ui <- function(id) {
   ns <- NS(id)
   bootstrapPage(
-    tags$h3(
-      textOutput(ns("message"))
+    div(
+      style = "display: flex; justify-content: center; align-items: center; height: 100vh;",
```

Comment:
I'm afraid this way we'll promote the not-so-good practice of using inline CSS. Summary of our options:
1. Leave the text unstyled (not pretty).
2. Edit `main.scss` and add `app.min.css` to the template (adds complexity / maintenance burden, more boilerplate).
3. Use inline CSS (promotes not-so-good practices).

I think (2) would be great if we had something like `rhino::init(plain = TRUE)` to omit all example code. We could then have even more example code - useful for Rhino newbies, not annoying for experienced users. WDYT - shall I add an issue to our Backlog?

For now, I'd personally go with (1). I'm hesitant to both (2) and (3) to similar degree, but not strongly. Use your own judgement :smile:

Reply:
I totally get your point, but the problem I have with (1) is that it looks just awful. I was thinking about having something that is more in a style of e.g. new React app. And I have no idea what will be the best choice here...

Reply:
Yup, I also thought about Create React App, they have a pretty nice initial app :+1:

I'm fine with this solution; please update `test-main.R` (so `rhino::test_r()` passes for a freshly initialized app) and I'll approve.

Reply:
Updated, PTAL

Reply:
But I also thought about option 4 - to hide either styled div or the whole element inside a component exported by Rhino. WDYT?
```r
rhino_start_page <- function(...) {
  div(
    style = "display: flex; justify-content: center; align-items: center; height: 100vh;",
    ...
  )
```

Reply:
That's an interesting idea (we already have `rhino::rhinos` in similar spirit - for demo purpose) but I'm afraid it would be misused for centering content in general. I'd go with what we have now :slightly_smiling_face: